docs/templates/plugin.rst.j2

```diff
@@ -151,7 +151,7 @@ Parameters
                         </ul>
                     {% endif %}
                     {# Show default value, when multiple choice or no choices #}
-                    {% if value.default is defined and value.default not in value.choices %}
+                    {% if value.default is defined and value.default|lower not in value.choices %}
```

Comment:
And @webknjaz , here is the potential problem I mentioned, the value.default might be a list or some other type, so the lower filter may fail here. But this change run well for all existing docs now.

Reply:
Yeah, we need to think this throuh. and also I think there's an open PR somewhere modifying these lines with `is_list` check. You might want to check it out as well.